src/updaterDB.js

```diff
@@ -905,7 +56,7 @@ const main = async () => {
   for (let i = 0; i < projects.length; i++) {
     await getProjects(projects[i]);
   }
-  fs.writeFile("./src/score.json", JSON.stringify(score, null, 2), (err) => {
+  fs.writeFile("./public/score.json", JSON.stringify(score, null, 2), err => {
```

Comment:
Change this path to `../public/score.json`

Reply:
done